data/mods/gennext/abilities.ts

```diff
@@ -280,7 +280,7 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 	reckless: {
 		inherit: true,
 		onBasePower(basePower, attacker, defender, move) {
-			if (move.recoil || move.hasCustomRecoil || attacker.item === 'lifeorb') {
+			if (move.recoil || attacker.item === 'lifeorb') {
```

Comment:
This is a change in behavior, I dont know if its desirable. I dont think mindblown moves exist in Gen Next but this would provide a boost to Struggle which is perhaps not desired? The fact that life orb was also included meant me think this was meant to be more inclusive of all types of recoil moves

Reply:
It's minor enough that you can just implement whatever makes it easier to code.